warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -1079,8 +1079,13 @@ public void close() {
         
     }
     
+    @Override
```

Comment:
@ivakegg we are

Reply:
Ah, this file was collapsed in my view so I missed it....sorry.